packages/@webex/plugin-meetings/src/meeting-info/meeting-info-v2.ts

```diff
@@ -336,16 +336,29 @@ export default class MeetingInfoV2 {
       requestOptions.resource = 'meetingInfo';
     }
 
+    if (meetingId) {
```

Comment:
what about non-USM meetings (v1)? We still have customers using them, they will have getMeetingInfoReqResp() returning undefined